src/test/unit_tests/test_s3_bucket_policy.js

```diff
@@ -335,6 +337,100 @@ mocha.describe('s3_bucket_policy', function() {
         }));
     });
 
+    mocha.it('should not allow principal get object bucket policy with 2 statements: ' +
+        '(1) DENY principal by account ID (2) ALLOW account name as *', async function() {
+        if (!is_nc_coretest) this.skip(); // eslint-disable-line no-invalid-this
+        const policy = {
+            Statement: [{
+                    Sid: `Allow all s3 actions on bucket ${BKT_D} to all principals`,
```

Comment:
this policy statement is being re-used later, please put in a const

Reply:
It is not the same policy, there's one mutual statement (allow to all participants with `*`), I'll try to reuse it.

Reply:
I meant to the allow all statement, not the whole policy